pages/_slug/index.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <section class="container">
+    <div>
+      <h1 class="title">
+        <h1>{{ page.fields.pageTitle }}</h1>
+      </h1>
+
+      <p>{{ page.fields.pageDescription }}</p>
+
+      <div
+        v-for="section in page.fields.section"
+        :key="section"
+        class="banner"
+      >
+        <h2>{{ section.fields.description }}</h2>
+        <ul>
+          <ContentCard
+            v-for="card in section.fields.contentCards"
+            :key="card.sys.id"
+            :card-title="card.fields.cardTitle"
+            :content-source="card.fields.contentSource"
+            :url="card.fields.url"
+            :image-url="card.fields.image.fields.file.url"
+          />
+        </ul>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import ContentCard from '~/components/contentCard.vue';
+import {createClient} from '~/plugins/contentful.js';
+
+const contentfulClient = createClient();
+
+export default {
+  asyncData ({params}) {
+    return Promise.all([
+      // fetch the browsePage data
+      contentfulClient.getEntries({
+        'sys.id': params.slug,
+        'include': 2 // this retrieves nested/linked resources like cards and their images
+      })
+    ]).then(([pages]) => {
+      // return data that should be available
+      // in the template
+      return {
+        page: pages.items[0]
+      };
+    }).catch(console.error);
```

Comment:
We don't want to use console, and I don't think this use of it would do anything anyway.